src/components/Recommended.tsx

```diff
@@ -40,12 +40,12 @@ const dates = [
   },
 ]
 
-const qualities = [
-  { id: 1, name: 'Medium', unavailable: false },
-  { id: 2, name: '4D', unavailable: false },
-  { id: 3, name: 'HD', unavailable: false },
-  { id: 4, name: 'Standard', unavailable: true },
-  { id: 5, name: 'Low', unavailable: false },
+const genres = [
```

Comment:
I think this data should be fetched from an the backend API

Reply:
@G-anna89 I can see where you are getting the categories from the API. I will leave this PR open so that you can push more commits to address that.